contracts/andromeda_factory/src/contract.rs

```diff
@@ -198,12 +219,32 @@ pub fn add_update_code_id(
 }
 
 #[cfg_attr(not(feature = "library"), entry_point)]
-pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> Result<Binary, ContractError> {
+pub fn query(deps: Deps, env: Env, msg: QueryMsg) -> Result<Binary, ContractError> {
     match msg {
         QueryMsg::GetAddress { symbol } => encode_binary(&query_address(deps, symbol)?),
-        QueryMsg::ContractOwner {} => encode_binary(&query_contract_owner(deps)?),
         QueryMsg::CodeId { key } => encode_binary(&query_code_id(deps, key)?),
-        QueryMsg::IsOperator { address } => encode_binary(&query_is_operator(deps, &address)?),
+        QueryMsg::AndrQuery(msg) => handle_andromeda_query(deps, env, msg),
+    }
+}
+
+fn handle_andromeda_query(
+    deps: Deps,
+    env: Env,
+    msg: AndromedaQuery,
+) -> Result<Binary, ContractError> {
+    match msg {
+        AndromedaQuery::Get(data) => {
+            let received: QueryMsg = parse_message(data)?;
```

Comment:
Do we maybe change the default Receive value to be a string, referencing the ADO for which to query the code id?

Reply:
Now that you mention that I actually did that in the cw20_hooks branch already.